### PR TITLE
feat: attached virtual and location to event creation flow

### PR DESCRIPTION
**Pull request by sieuhuflit**
## Overview
- Provide edit virtual when create event
- Can attached location into event creation flow

## Demo

Dec-21-2023 12-06-59

### CODE REVIEW

File: lib/core/application/event/create_event_bloc/create_event_bloc.dart [hunk 1]
```diff
@@ -124,24 +124,37 @@ class CreateEventBloc extends Bloc<CreateEventEvent, CreateEventState> {
...
-          ),
-          virtual: true,
+      var input = Input$EventInput(
+        title: title.value,
+        description: title.value,
+        private: false,
+        verify: false,
+        start: DateTime.parse(event.start.toUtc().toIso8601String()),
+        end: DateTime.parse(event.end.toUtc().toIso8601String()),
+        timezone: "Asia/Bangkok",
```
Comment (mthinh) [inline lib/core/application/event/create_event_bloc/create_event_bloc.dart:134]: why timezone is hardcoded ?

Comment (sieuhuflit) [reply lib/core/application/event/create_event_bloc/create_event_bloc.dart]: Resolved

**Comment by sieuhuflit** [comment]
Please help review again 🙏 @mthinh

**Comment by mthinh** [comment]
flutter_native_timezone is outdated, 2 years no maintained, please try another package which is well maintained. 
Suggest https://pub.dev/packages/flutter_timezone

**Review by mthinh — CHANGES_REQUESTED** [review]
Leave one change request

**Comment by sieuhuflit** [comment]
> flutter_native_timezone is outdated, 2 years no maintained, please try another package which is well maintained. Suggest https://pub.dev/packages/flutter_timezone

holy, I'm aware about this, but I install the wrong package 😆

**Comment by sieuhuflit** [comment]
Resolved @mthinh please help re-check

**Review by mthinh — APPROVED** [review]

**Pull request merged**